src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -59,12 +67,31 @@ html, body {
  * scaled to allow button display
  */
 #map {
-    height: 80%;
+    height: 90%;
     width: 100%;
     margin: auto;
-    position: relative;
-    top: 170px;
+    top: 180px;
     -ms-transform: translateY(-50%);
     transform: translateY(-50%);
-    overflow: inherit !important;
+    zoom: 1.15;
+}
+
+.gm-style-mtc {
```

Comment:
just curious -- are these documented in the api and they say to style the map this way, or did you just inspect to see what the class values were?